runtime/testy/stateManager.py

```diff
@@ -152,9 +156,10 @@ def student_upload(self):
     self.processMapping[PROCESS_NAMES.TCP_PROCESS].send([ANSIBLE_COMMANDS.STUDENT_UPLOAD, True])
 
   def send_console(self, console_log):
+    return
```

Comment:
This is deliberate to enable console logging to not crash the robot. see pull request message for details.